### PR TITLE
various tweaks

### PR DESCRIPTION
**Pull request by PromoFaux**
Signed-off-by: Adam Warner <me@adamwarner.co.uk>

### CODE REVIEW

File: docs/guides/whitelist-blacklist.md [hunk 1]
```diff
@@ -4,11 +4,9 @@
 
 ### How to Whitelist or Blacklist
 
-There are scripts to aid users in adding or removing domains to the whitelist or blacklist.
+There are scripts to aid users in adding or removing domains to the whitelist or blacklist from the CLI
```
Comment (XhmikosR) [inline docs/guides/whitelist-blacklist.md:7]: Missing period

File: docs/guides/whitelist-blacklist.md [hunk 2]
```diff
@@ -4,11 +4,9 @@
...
 
-The scripts will first parse `whitelist.txt` or `blacklist.txt` for any changes, and if any additions or deletions are detected, it will reload `dnsmasq` so that they are effective immediately.
-
-Each script accepts the following parameters:
+Each script accepts the following parameWters:
```
Comment (XhmikosR) [inline docs/guides/whitelist-blacklist.md:9]: typo `parameWters`

Comment (PromoFaux) [reply docs/guides/whitelist-blacklist.md:9]: How in the hell did i manage that?

File: docs/core/pihole-command.md [hunk 3]
```diff
@@ -339,4 +320,4 @@ Help Command    | `pihole -a interface --help`
 Script Location | [`/opt/pihole/webpage.sh`](https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/webpage.sh)
 Example Usage   | [`pihole -a interface local`](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#interface)
 
-Specify interface listening behavior for `dnsmasq`. When using `pihole -a interface all`, please ensure you use a firewall to prevent your Pi-hole from becoming an unwitting host to [DNS amplification attackers](https://duckduckgo.com/?q=dns+amplification+attack). You may want to consider running [OpenVPN](https://github.com/pi-hole/pi-hole/wiki/Pi-hole---OpenVPN-server) to grant your mobile devices access to the Pi-hole.
+Specify interface listening behavior for `pihole-FTL`. When using `pihole -a interface all`, please ensure you use a firewall to prevent your Pi-hole from becoming an unwitting host to [DNS amplification attackers](https://duckduckgo.com/?q=dns+amplification+attack). You may want to consider running [OpenVPN](https://github.com/pi-hole/pi-hole/wiki/Pi-hole---OpenVPN-server) to grant your mobile devices access to the Pi-hole.
```
Comment (XhmikosR) [inline docs/core/pihole-command.md:323]: @PromoFaux Don't we have a docs page we could link to for OpenVPN instead of the Wiki?

Comment (PromoFaux) [reply docs/core/pihole-command.md:323]: We absolutely do. Within the guides section of this very repo :)

Comment (XhmikosR) [reply docs/core/pihole-command.md:323]: Can you link to that please? :)

Comment (PromoFaux) [reply docs/core/pihole-command.md:323]: /guides/vpn/overview

**Comment by XhmikosR** [comment]
Done. Branch rebased and also fixed links to be relative like everywhere else.

**Pull request merged**